src/gz21_ocean_momentum/step/data/lib.py

```diff
@@ -39,48 +185,22 @@ def advections(u_v_field: xr.Dataset, grid_data: xr.Dataset):
     adv_x = u * gradient_x["usurf"] + v * gradient_y["usurf"]
     adv_y = u * gradient_x["vsurf"] + v * gradient_y["vsurf"]
     result = xr.Dataset({"adv_x": adv_x, "adv_y": adv_y})
-    # TODO check if we can simply prevent the previous operation from adding
-    # chunks
+    # TODO 2023-09-20: old note from original import: v
```

Comment:
Remove Todo and make Github issue? @arthurBarthe , does this comment say anything to anyone anymore?